### DISCUSSION TITLE
npx prisma init error

### DISCUSSION THREAD

**Discussion by KKaranja** [prisma/prisma#14936] [Q&A] [answered]
Am getting the error below when I try to initiate prisma, any idea why am getting the error?   
> Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/73e60b76d394f8d37d8ebd1f8918c79029f0db86/windows/migration-engine.exe.gz.sha256 failed, reason: 
unable to get local issuer certificate
error Command failed with exit code 1.

**Comment by janpio** [comment]
Any local network configuration that is uncommon? The file is available for download, so this seems to be something in your setup that leads to a message when checking the SSL certificate chain.

**Comment by Rankarusu** [comment, marked as answer]
I'm having the same issue as well when trying the command from behind a corporate proxy. Setting a ca-file in yarn/npm or as environment variable did not work.

**Reply by janpio** [reply]
DId you have a look at these? https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#proxy-environment-variables

**Reply by vladi-stevanovic** [reply]
👋 Hello @KKaranja and @Rankarusu  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open? Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Reply by Rankarusu** [reply]
> 👋 Hello @KKaranja and @Rankarusu I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open? Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!
> 
> It would be super useful for other users facing this problem! 🙌

Hi there! Thank you for your reply. The issue is solved for me. Local IT gave me a hand and added a rule to the proxy to not check that url for certificates. Works like a charm since then. I would assume that the issue still persists from prismas side.

**Comment by aryan0-hash** [comment]
Hi @Rankarusu @janpio @nurul3101  i'm also facing the same issue can someone help please i got struck in this since two week I have all the thing which is present in internet but still not working please help me?

**Reply by Jolg42** [reply]
@aryan0-hash Could you open a new discussion and give as much information as you can?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
